### PR TITLE
Fix compatibility with pytest dev/6.2

### PR DESCRIPTION
**Pull request by saimn**

### CODE REVIEW

File: pytest_doctestplus/plugin.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 _pytest_version = Version(pytest.__version__)
 PYTEST_GT_5 = _pytest_version > Version('5.9.9')
-PYTEST_GT_6_2 = _pytest_version > Version('6.2')
+PYTEST_GE_6_3 = _pytest_version >= Version('6.3')
```
Comment (pllim) [inline pytest_doctestplus/plugin.py:25]: This was what I actually started with, but somehow didn't work.

**Comment by saimn** [comment]
It gets more annoying ...
```
(Pdb) Version(pytest.__version__)
<Version('6.3.0.dev19+g950fbbc32')>
(Pdb) Version(pytest.__version__) >= Version('6.3')
False
```

File: pytest_doctestplus/plugin.py (hunk 1 above)
Comment (saimn) [reply pytest_doctestplus/plugin.py:25]: Yep, I see that :(

File: pytest_doctestplus/plugin.py [hunk 2]
```diff
@@ -22,7 +22,7 @@
 
 _pytest_version = Version(pytest.__version__)
 PYTEST_GT_5 = _pytest_version > Version('5.9.9')
-PYTEST_GE_6_3 = _pytest_version >= Version('6.3')
+PYTEST_GE_6_3 = _pytest_version.is_devrelease or _pytest_version >= Version('6.3')
```
Comment (pllim) [inline pytest_doctestplus/plugin.py:25]: Or something hacky like this if this doesn't work... 😬 
```python
PYTEST_GT_6_2 = _pytest_version >= Version('6.2.99')
```

Comment (saimn) [reply pytest_doctestplus/plugin.py:25]: It works ;), so maybe good enough for now ?

**Review by pllim — APPROVED** [review]
Thanks for cleaning up my mess!

**Comment by saimn** [comment]
My pleasure ;)

**Pull request merged**